alumnium/models.py

```diff
@@ -9,6 +9,7 @@ class Provider(Enum):
     AWS_META = "aws_meta"
     DEEPSEEK = "deepseek"
     GOOGLE = "google"
+    MISTRAL_AI = "mistral_ai"
```

Comment:
I think the convention is to use MistralAI as a single word

```suggestion
    MISTRALAI = "mistralai"
```